Makefile

```diff
@@ -100,7 +100,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	cd config/deployment && $(KUSTOMIZE) edit set image controller=${IMG}
```

Comment:
the image name [here](https://github.com/stolostron/hypershift-deployment-controller/blob/main/config/deployment/template-deployment.yaml#L20) is `quay.io/stolostron/hypershift-deployment-controller:latest` instead of `controller`, so i think this should be: 
`cd config/deployment && $(KUSTOMIZE) edit set image quay.io/stolostron/hypershift-deployment-controller:latest=${IMG}`